reasoner/controller/ConcludableController.java

```diff
@@ -158,7 +159,7 @@ protected Processor.Explain createProcessorFromDriver(Driver<Processor.Explain>
             //  concludable. They should be filtered before being passed to the concludableProcessor's constructor
             assert bounds.equals(this.bounds);
             return new Processor.Explain(
-                    explainDriver, driver(), processorContext(), bounds, set(), conclusionUnifiers, reasonerConsumer,
+                    explainDriver, driver(), processorContext(), concludable, bounds, set(), conclusionUnifiers, reasonerConsumer,
```

Comment:
this should have a comment as to why we ned this new argument

Reply:
True - we needed this argument to be able to check whether the concludable fits the criteria that mean that may have received a non-inferred fact from a conclusion to answer this concludable.